app/src/main/java/org/mifos/mobile/ui/beneficiary/presentation/beneficiaryaddoptionsscreen.kt

```diff
@@ -0,0 +1,89 @@
+package org.mifos.mobile.ui.beneficiary.presentation
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.mifos.mobile.R
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+
+
+/**
+ * This is a composable function that represent the beneficiary screen and the peremeters include
+ * @param[addiconClicked] to navigate to the add screen
+ * @param[scaniconClicked] to navigate to the scanicon screen
+ * @param[uploadiconClicked] to navigate to the upload icin screen
+ **/
+
+
+@Composable
+fun beneficiaryscreen(
```

Comment:
rename it to `BeneficiaryScreen` . We are following a convention so adhere to that

Reply:
I have made this corrections